lib/src/platform_client.dart

```diff
@@ -1089,6 +1089,19 @@ class PlatformClient {
     return ChatMessageInfo.fromJson(response);
   }
 
+  /// Sends user feedback on the AI response.
+  Future<void> sendMessageFeedback(int sessionId, int messageId, int rating, String comment) async {
```

Comment:
For consistency with the other chat APIs, let's rename the method to `sendChatMessageFeedback` (to distinguish from other "messages") and rename `sessionId` to `chatId`.

The other feedback method in this class is `saveFeedback`. I'm not sure if I like `save` or `send` (or something else) better, so I'm fine leaving it `send` for now.